model/schema/gene.yaml

```diff
@@ -86,3 +94,31 @@ slots:
     description: SOTerm describing gene type
     domain: Gene
     range: SOTerm
+
+  gene_designating_laboratory:
+    description: >-
+      A laboratory which designated this gene
+    required: false
+    multivalued: false
+    domain: GeneClass
+    range: Laboratory
+
+  designating_person:
```

Comment:
For multivalue slots we tend to pluralise the slot name, i.e. designating_persons

Reply:
done